src/System.Data.Common/tests/System/Data/DataTableTest.cs

```diff
@@ -885,9 +885,10 @@ public void PropertyExceptions()
 
                 Assert.Throws<ArgumentException>(() =>
                 {
-                    CultureInfo cultureInfo = new CultureInfo("en-gb");
+                    // Set to a different culture than before: this breaks the DataRelation constraint
+                    // because it is not the locale of the related table
+                    CultureInfo cultureInfo = table.Locale.Name == "en-US" ? new CultureInfo("en-GB") : new CultureInfo("en-US");
                     table.Locale = cultureInfo;
-                    table1.Locale = cultureInfo;
```

Comment:
You've removed this because the first assignment to table.Locale throws and so we never get here?  Is that not the case in the Assert.Throws for CaseSensitive above this?  Seems like either that `table1.CaseSensitive = true` should also be deleted if the `table.CaseSensitive = true` throws, or the `table.CaseSensitive = true` should be moved out of and to before the Assert.Throws, so that we're validating exactly what's supposed to throw.

Reply:
OK, done.